tests/metrics/run_docker_metrics

```diff
@@ -1,24 +1,19 @@
 #!/bin/bash
 
 source ./run_docker_metrics.dat
+
+# Verify that test-common.bash is generated.
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+[ -f "${SCRIPT_PATH}/../lib/test-common.bash" ] || \
```

Comment:
Hi. Can you double check this file check line for me. It seems to always exit on my system (ubuntu under ciao-down) and doesn't run the tests (when invoked with the `make metrics-tests` from the top level dir).
To check I ran the script in the subdir with -x. Output:

```
gwhaley@singlevm:/work/gwhaley/gopath/src/github.com/01org/cc-oci-runtime/tests/metrics$ sudo bash -x ./run_docker_metrics
+ source ./run_docker_metrics.dat
++ TIMES=100
++ MEM_CONTAINERS=10
++ MEM_WAIT_TIME=30m
++ CPU_WAIT_TIME=2m
+++ readlink -f ./run_docker_metrics
++ dirname /work/gwhaley/gopath/src/github.com/01org/cc-oci-runtime/tests/metrics/run_docker_metrics
+ SCRIPT_PATH=/work/gwhaley/gopath/src/github.com/01org/cc-oci-runtime/tests/metrics
+ '[' -f /work/gwhaley/gopath/src/github.com/01org/cc-oci-runtime/tests/metrics/../lib/test-common.bash ']'
+ exit 1
gwhaley@singlevm:/work/gwhaley/gopath/src/github.com/01org/cc-oci-runtime/tests/metrics$ ls ../lib
test-common.bash  test-common.bash.in  test-crio.bats  test-crio.bats.in
```

I also checked - it does print the warning and exit when the tests-common.bash file is removed.

I'd probably just open-code the line as an if/fi line :-)